horizons/component/componentholder.py

```diff
@@ -180,6 +180,6 @@ def get_component_template(cls, component):
 		for entry in cls.component_templates:
 			if isinstance(entry, dict):
 				for key, value in entry.iteritems():
-					if cls.class_mapping[key] == component or key == component:
+					if component in [cls.class_mapping[key], key]:
```

Comment:
This feels a bit weird, since you're switching the operands of the comparison. Which is syntactically fine of course, but in this case it hurts readability in my opinion. We're iterating over the items in `entry`, and check whether the key equals the component. This change feels like doing `2 == x` instead of `x == 2`.